src/index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" media="screen" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header-container">
+            <nav class="nav-item1"><a class="Mylogo" href="">My Logo</a></nav>
+            <nav class="nav-item2"></nav>
+            <nav class="nav-item3">
+                <img class="toggle" src="../assets/toggle.png" alt="toggle" onclick="openNav()">
+            </nav>
+            <div id="myNav" class="overlay">
+                <a class="closebtn"> &times; </a>
+                <div class="overlay-content">
+                    <a class="pop-nav" href="#portfolio">Portfolio</a>
+                    <a class="pop-nav" href="#about">About</a>
+                    <a class="pop-nav" href="#contact">Contact</a>
+                </div>
+            </div>
+        </div>
+        <nav class="web-nav">
+            <div class="logo">
+                <a class="Mylogo" href="">My Logo</a>
+            </div>
+            <ul>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="content">
+        <h1 class="intro">I’m Daniel Glad to see you!</h1>
+        <p class="intro1">
+            I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a class="intro2">LET’S CONNECT</a>
+        <ul class="social">
+            <li class="social-icon twitter">
+                <a><img src="../assets/twitter.png" alt="twitter"></a>
+            </li>
+            <li class="social-icon linkedin">
+                <a><img src="../assets/Linkedin.png" alt="linkedin"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="../assets/m.png" alt="m"></a>
+            </li>
+            <li class="social-icon github">
+                <a><img src="../assets/github.png" alt="github"></a>
+            </li>
+            <li class="social-icon fingers">
+                <a><img src="../assets/fingers.png" alt="fingers"></a>
+            </li>
+        </ul>
+    </section>
+    <section id="portfolio" class="project-section">
+
+    </section>
+    <section id="about" class="about">
```

Comment:
- [ ] I kindly suggest that you follow the portfolio design in making your about page:
- Make sure that you have a top right border radius as seen in the Figma design.
- Make sure that your text is properly aligned as seen in the Figma design
- Kindly ensure that `About` and `myself` are not on the same line.


| Your Design | Figma Design |
| ------------- | ------------- |
| <img width="308" alt="image" src="https://user-images.githubusercontent.com/52934128/220825113-fb77eb7b-e658-4141-ae92-e91a83f471ed.png">  | <img width="282" alt="image" src="https://user-images.githubusercontent.com/52934128/220824481-416871e2-7f9e-4a50-b599-fa68a05c9ed6.png">  |
| Content Cell  | Content Cell  |